apps/frontend/src/libs/hooks/use-search/use-search.hook.ts

```diff
@@ -1,20 +1,26 @@
 import { QueryParameterName } from "~/libs/enums/enums.js";
 import { useEffect, useSearchParams, useState } from "~/libs/hooks/hooks.js";
 
-const useSearch = (): {
+const useSearch = (
+	useQueryParameters: boolean = true,
+): {
```

Comment:
I think it's better to pass an argument as an object, useSeatch(false) doesn't look very clear. 
And don't forget about B19 (now naming is the same as hooks)